docs/API/content-type/creating-co.md

```diff
@@ -3,106 +3,392 @@ description: How to add Content Objects in Flotiq
 
 # Content Objects
 
-Once a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> has been defined in the system - the user can create <abbr title="Content Object - an instance of a Content Type.">Content Objects</abbr>  of that Content Type. This is done either directly through the API or via the convenient Content Entry tools provided within the Content Management Platform.
+Once a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> 
+has been defined in the system - the user can create 
+<abbr title="Content Object - an instance of a Content Type.">Content Objects</abbr>  of that Content Type. 
+This is done either directly through the API or via the convenient Content Entry tools provided within the 
+Content Management Platform.
 
-## Authoring content through the API
+!!! note
+    You will need to use your `Application Read and write API KEY` to perform this action
+    or `User API KEY` scoped to accept create on the Content Type you wish to add.
+    Read more about [API keys and scoped API keys](/API/).
 
-The supporting endpoints of a given <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> allow the user to perform basic REST operations 
+## Creating Content Objects through the API
 
-![](../images/endpoints.png){: .center .width75 .border}
+For a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> 
+defined according to the [create Content Type example](/API/content-type/creating-ctd), a very simple `POST` payload can be sent 
+to the supporting endpoint `https://api.flotiq.com/api/v1/content/{name}` 
+(where `name` is the name of the content type definition) to create a new Content Object:
 
-
-For a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> defined according to the example described above, a very simple POST payload can be sent to the supporting endpoint to create a new Content Object:
-
-``` json
+```
 {
-  "id": "123123123",
   "title": "New object",
```

Comment:
Removing id affects next examples on batch (updateExisting) which presents update based on `id`; I will add "Let's assume that we have blogpost with id `123123123123` to the batch section.